src/checkbox/styled-components.js

```diff
@@ -189,20 +191,20 @@ export const Checkmark = styled('span', props => {
     marginBottom: $theme.sizing.scale0,
     marginLeft: $theme.sizing.scale0,
     marginRight: $theme.sizing.scale0,
-  };
+  }: {});
```

Comment:
Is there a reason we need to type it here but not other styled fns?

Reply:
it's because `null` is not valid [here](https://github.com/uber-web/baseui/pull/1241/files/1ab536dfc0376d74483df83699da7111ef1abff7#diff-5d4eec77dcb85451082356a170ef0e24R182). this coerces the type